README.md

```diff
@@ -4,19 +4,11 @@ Network boot and provision Container Linux clusters on virtual or physical hardw
 
 **Announcement**: A new [Matchbox Terraform Provider](https://github.com/coreos/terraform-provider-matchbox) is available for Terraform users.
 
-## Guides
-
-* [Network Setup](Documentation/network-setup.md)
-* [Machine Lifecycle](Documentation/machine-lifecycle.md)
-* [Background: PXE Booting](Documentation/network-booting.md)
-* Tutorials (QEMU/KVM/libvirt)
-    * [matchbox with rkt](Documentation/getting-started-rkt.md)
-    * [matchbox with Docker](Documentation/getting-started-docker.md)
-
 ## matchbox
 
-`matchbox` is an HTTP and gRPC service that renders signed [Ignition configs](https://coreos.com/ignition/docs/latest/what-is-ignition.html), [cloud-configs](https://coreos.com/os/docs/latest/cloud-config.html), network boot configs, and metadata to machines to create Container Linux clusters. Groups match machines based on labels (e.g. MAC, UUID, stage, region) and use named Profiles for provisioning. Network boot endpoints provide PXE, iPXE, and GRUB. `matchbox` can be deployed as a binary, as an [appc](https://github.com/appc/spec) container with [rkt](https://coreos.com/rkt/docs/latest/), or as a Docker container.
+`matchbox` is a service that matches machines (based on labels like MAC, UUID, etc.) to profiles to PXE boot and provision Container Linux clusters. Profiles specify the kernel/initrd, kernel arguments, iPXE config, GRUB config, [Container Linux Config](Documentation/container-linux-config.md), [Cloud-Config](Documentation/cloud-config.md), or other configs a machine should use. Matchbox can be [installed](Documentation/deployment.md) as a binary, RPM, container image, or deployed on a Kubernetes cluster and it provides an authenticated gRPC API for clients like [terraform](https://github.com/coreos/terraform-provider-matchbox).
```

Comment:
I can for this paragraph. The page itself is almost entirely links so I dunno if it makes sense for the bullet points.